pkg/server/container_create.go

```diff
@@ -481,7 +481,19 @@ func addOCIBindMounts(g *generate.Generator, mounts []*runtime.Mount, mountLabel
 	for _, mount := range mounts {
 		dst := mount.GetContainerPath()
 		src := mount.GetHostPath()
-		options := []string{"rbind", "rprivate"}
+		options := []string{"rbind"}
+		switch mount.GetPropagation() {
+		case runtime.MountPropagation_PROPAGATION_PRIVATE:
+			options = append(options, "rprivate")
+		case runtime.MountPropagation_PROPAGATION_BIDIRECTIONAL:
+			options = append(options, "rshared")
+		case runtime.MountPropagation_PROPAGATION_HOST_TO_CONTAINER:
+			options = append(options, "rslave")
+		default:
+			glog.Warningf("unknown propagation mode for hostPath %q", mount.HostPath)
```

Comment:
nit: s/unknown/Unknown

Log should start with capitalized letter. :)

Reply:
@Random-Liu  Thank for review, Done.